Rules/CommonScripts/accolade_data.cfg

```diff
@@ -1973,6 +2132,10 @@ Skinney =
 	# community - map moderator
 	# map - map moderator
 
+Stolij =
```

Comment:
Hi, thank you for awarding the Github Accolade to my account. There is only one small issue: My King Arthur's Gold account name is all lower case, so i believe it should be ```stolij``` instead of ```Stolij```

Reply:
thanks, my bad, updated